ida-ws/src/main/java/upb/ida/logging/AspectLogger.java

```diff
@@ -0,0 +1,145 @@
+package upb.ida.logging;
+
+/**
+ * Logs Creation - Using Spring AOP to provide universal logging of all the
+ * exceptions and method calls.
+ * 
+ * @author Ayaz
+ *
+ */
+
+import javax.servlet.http.HttpServletRequest;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import upb.ida.bean.ResponseBean;
+import upb.ida.constant.IDALiteral;
+import org.apache.log4j.BasicConfigurator;
+import org.apache.log4j.Logger;
+
+@Component
+@Aspect
+public class AspectLogger 
+{
+	@Autowired(required=true)
+	private HttpServletRequest request;
+	@Autowired
+	private ResponseBean response;
+	@Autowired
+	@Qualifier("requestResponseLogger")
+	Logger reqRespLogger;
+	@Autowired
+	@Qualifier("exceptionLogger")
+	Logger exceptionLogger;
+	@Autowired
+	@Qualifier("datasetResponseLogger")
+	Logger datasetResponseLogger;
+	
+    /**
+     * Constructor of AspectLogger class
+     */
+	AspectLogger()
+	{
+		BasicConfigurator.configure();
```

Comment:
could you please confirm once by removing this statement and checking if its really needed?
if yes, then put it in a static block inside your loggerprovider class